src/Entity/CitizenProject.php

```diff
@@ -385,4 +387,30 @@ public function getCreator(): ?Adherent
     {
         return $this->creator;
     }
+
+    public function getSkills(): Collection
```

Comment:
It's ok (even better) to keep phpdocs which bring some actual value like `CitizenProjectSkill[]|Collection`.